packages/auth/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPlugin.kt

```diff
@@ -103,6 +103,25 @@ open class AmplifyAuthCognitoPlugin :
     )
   }
 
+  /**
+   Legacy Device Secrets Storage.
+   */
+  private val legacyDeviceSecretsStore: LegacyKeyValueStore by lazy {
+    LegacyKeyValueStore(
+      applicationContext!!,
+      "CognitoIdentityProviderDeviceCache.<userPoolId>.<username>"
```

Comment:
This doesn't look correct to me. `<userPoolId>` and `<username>` probably need to be replaced with the actual value for the userPoolId and username.

Reply:
Maybe this is unused? I see a LegacyKeyValueStore instance in fetchLegacyDeviceSecrets with a similar name.